File: Logic_HandsOn/C++/Problems on Digits,String,Array,Pointers etc/program32.cpp
///////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept number from user and display below pattern.
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<iostream>
using namespace std;

class Ans
{
	private:
		int iNo;
		int iCnt;
		int iVal;

	public:
		Ans(int iNo,int iCnt=0)
		{
			this->iNo = iNo;
			this->iVal = 65;
		}

		void Display()
		{
			for(iCnt=5;iCnt>=1;iCnt--)
    			{
				cout<<iCnt<<"   #\t";
          
    			}

			cout<<"\n";
		}
};

int main()
{
	int iValue=0;

	cout<<"Enter number of elements: ";
	cin>>iValue;
	
	Ans *aObj = new Ans(iValue);
	aObj->Display();
}

/*
OUTPUT:

Enter number of elements: 5
5   #	4   #	3   #	2   #	1   #

*/

File: Logic_HandsOn/Java/Problems on Array,Pattern etc/Program32.java
///////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Accept number from user and display below pattern.
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Solution
{
	public int iValue;
	public int iCnt;

	public Solution(int iValue)
	{
		this.iValue = iValue;
		this.iCnt=0;
	}

	public void Display()
	{
		for(iCnt=5;iCnt>=1;iCnt--)
		{
			System.out.print(iCnt+"   #\t"); 
		}

		System.out.println();
	}
}

class Program2
{
	public static void main(String[] args)
	{
		Scanner sObj = new Scanner(System.in);

		System.out.println("Enter number of elements:");
		int iNo = sObj.nextInt();

		Solution obj = new Solution(iNo);
		obj.Display();
	}
}

/*
OUTPUT:

Enter number of elements:
2
5   #	4   #	3   #	2   #	1   #

*/
